meshroom/core/desc/node.py

```diff
@@ -71,6 +71,14 @@ class BaseNode(object):
             value="",
             invalidate=False,
         ),
+        ChoiceParam(
+            name='nodeDefaultLogLevel',
+            label='Default logging level',
```

Comment:
```suggestion
            label='Default Logging Level',
```
For labels, we're trying to always use PascalCase.